### PR TITLE
Remove unsupported 1.18 jobs

### PR DESCRIPTION
**Pull request by xing-yang**

### CODE REVIEW

File: config/jobs/kubernetes-csi/external-provisioner/external-provisioner-config.yaml [hunk 1]
```diff
@@ -2,102 +2,6 @@
 
 presubmits:
   kubernetes-csi/external-provisioner:
-  - name: pull-kubernetes-csi-external-provisioner-1-18-on-kubernetes-1-18
```
Comment (msau42) [inline config/jobs/kubernetes-csi/external-provisioner/external-provisioner-config.yaml:5]: 1.18 should be removed from gen-jobs.sh, and then all the files should be updated.

Comment (xing-yang) [reply config/jobs/kubernetes-csi/external-provisioner/external-provisioner-config.yaml:5]: ok

**Pull request merged**